src/components/Breadcrumbs/Breadcrumbs.stories.tsx

```diff
@@ -12,9 +12,9 @@ export default {
 
```

Comment:
Do we want that `subcomponents` part from line 10 in the other components too?

Reply:
ooh yeah that documentation is nice. will update